packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -36,6 +36,23 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    * @default - Health check configuration from container.
    */
   readonly healthCheck?: HealthCheck;
+
+  /**
+   * The minimum number of CPU units to reserve for the container.
+   *
+   * @default - No minimum CPU units reserved.
+   */
+  readonly containerCpu?: number;
+
+  /**
+   * The amount (in MiB) of memory to present to the container.
+   *
+   * If your container attempts to exceed the allocated memory, the container
+   * is terminated.
+   *
+   * @default - No memory limit.
+   */
+  readonly containerMemoryLimitMiB?: number;
```

Comment:
To understand it better for review, seeing the original issue i think there was a suggestion that we could use task definition of fargate to spcify cpu and memory limits, is that not possible for this use case..?

Reply:
@shikha372 

When specifying `taskDefinition` as a props of `ApplicationLoadBalancedFargateService`, the newly added `containerCpu` and `containerMemoryLimitMiB` props in this MR do not work.

See: https://github.com/aws/aws-cdk/blob/89290c648a2234479802e1218b6dd2e47f7b1b33/packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts#L87-%23L99

so, it seems impossible.